tutorials/testing-coreutils.md

```diff
@@ -61,108 +61,113 @@ src$ rm -f *.gcda # Get rid of any stale gcov files
 src$ ./echo**
 
 src$ ls -l echo.gcda
--rw-r--r-- 1 ddunbar ddunbar 1832 2009-08-04 21:14 echo.gcda
+-rw-rw-r-- 1 klee klee 896 Nov 21 22:00 echo.gcda
 src$ gcov echo
+File '../../src/echo.c'
+Lines executed:24.27% of 103
+Creating 'echo.c.gcov'
+
 File '../../src/system.h'
-Lines executed:0.00% of 47
-../../src/system.h:creating 'system.h.gcov'
+Lines executed:0.00% of 3
+Creating 'system.h.gcov'
 
-File '../../lib/timespec.h'
-Lines executed:0.00% of 2
-../../lib/timespec.h:creating 'timespec.h.gcov'
+{% endhighlight %}
 
-File '../../lib/gettext.h'
-Lines executed:0.00% of 32
-../../lib/gettext.h:creating 'gettext.h.gcov'
+By default `gcov` will show the number of lines executed in the program (the `.h` files include code which was compiled into `echo.c`).
 
-File '../../lib/openat.h'
-Lines executed:0.00% of 8
-../../lib/openat.h:creating 'openat.h.gcov'
+## Step 2: Install WLLVM
 
-File '../../src/echo.c'
-Lines executed:18.81% of 101
-../../src/echo.c:creating 'echo.c.gcov'
+One of the difficult parts of testing real software using KLEE is that it must be first compiled so that the final program is an LLVM bitcode file and not a linked executable. The software's build system may be set up to use tools such as 'ar', 'libtool', and 'ld', which do not in general understand LLVM bitcode files.
```

Comment:
s/linked executable/native binary/

The LLVM bitcode file we produced is linked and it could be argued that it is an "executable" in some sense. Saying "native binary" might be clearer.